### PR TITLE
[chore] use bigger runner for load tests

### PR DESCRIPTION
**Pull request by mowies**
This PR changes the runner for our load tests to our new github hosted larger runner with 16 core.

### CODE REVIEW

File: .github/workflows/load-tests.yml [hunk 1]
```diff
@@ -57,7 +57,7 @@ jobs:
           path: ./bin/*
 
   loadtest:
-    runs-on: ubuntu-24.04
+    runs-on: "Linux-x64-16-64GB-Runner"
```
Comment (arminru) [inline .github/workflows/load-tests.yml:60]: If we know the image version (ubuntu-latest or ubuntu-24.04?) we could document it in a comment.

Comment (mowies) [reply .github/workflows/load-tests.yml]: done

**Comment by odubajDT** [comment]
what about the combined scenario? Do you think we need a bigger runner for that as well?

**Comment by mowies** [comment]
good point! yes we should also use the runner there

**Comment by mowies** [comment]
the combined load test is now also moved to the larger runner and it only runs on pushes to main branch same as for the rest of the load tests

**Review by bacherfl — APPROVED** [review]

**Pull request merged**